src/Framework/Config/ConfigLoader.php

```diff
@@ -107,6 +111,25 @@ private function readLocalConfigFile(GacelaConfigFile $gacelaConfigFile): array
         return array_merge(...array_filter($result));
     }
 
+    private function generateAbsolutePathWithSuffix(string $relativePath): string
+    {
+        // place the file suffix right before the file extension
```

Comment:
What about `.env` files? Did you already take them into account? 😄

Reply:
Also, the order (when `APPLICATION_ENV=env`) is the following?

1. `default.php`
2. `default-dev.php`
3. defined in the `gacela.php` under `path_local` key, e.g: `local.php`

Reply:
Actually, that's the desired behaviour, yes! But I think it doesn't work right now like that... I need to refactor and extract this functionality `generateAbsolutePathWithSuffix` on its own class, so I can rite unit tests for each behaviour. Thanks for spotting this.

Also, I had an idea of allowing the user to define different config-name-with-env merging strategies that will make this feature totally customisable and extensible. This will naturally come afterwards.